styling/src/jsMain/kotlin/dev.fritz2.styling/theme/definitions.kt

```diff
@@ -663,3 +660,22 @@ interface ToastStatus {
 interface ToastButton {
     val close: Style<BasicParams>
 }
+
+/**
+ * definition of the theme's appFrame
+ */
+interface AppFrameStyles {
+    val headerHeight: Property
+    val footerMinHeight: Property
+    val mobileSidebarWidth: Property
+    val brand: Style<FlexParams>
+    val sidebar: Style<BasicParams>
+    val nav: Style<BasicParams>
+    val footer: Style<BasicParams>
+    val header: Style<FlexParams>
+    val main: Style<BasicParams>
+    val tabs: Style<FlexParams>
+    val navLink: Style<FlexParams>
+    val navSection: Style<BasicParams>
+}
+
```

Comment:
How about separating the navLink component styles from the appFrame by extracting a separate interface within the theme and just refering to it by the ``AppFrameStyles``? (This has allready been used for ``SeverityAware`` as most forms are styled in exactly the same way)

This approach has the following benefits:
- does not mix different components and their stylings
- does not close the possibility to separate the styling between appFrame and content navigation - even though this seems to be an unusual use case

The added complexity is rather small imho.

Reply:
navLink and navSection should not be used outside off appFrame because of their responsive behavior that does not make sense elsewhere. added this to the docs